### PR TITLE
Refactored code and changed pins to use #define

### PR DESCRIPTION
**Pull request by Vlad-Galeev**

### CODE REVIEW

File: firmware_no_aux.cpp [hunk 1]
```diff
@@ -21,196 +23,190 @@ Adafruit_NeoPixel pixels(NUMPIXELS, PIN, NEO_GRB + NEO_KHZ800);
...
+    while (*str)
+        send(*str++);
+    Serial.println("");
 }
 
 
-//Radio shtuff
-RH_ASK driver(2000, 7, 12, 5); //Which will initialise the driver at 2000 bps, recieve on GPIO7, transmit on GPIO12, PTT on GPIO5. This is very poorly documented. PTT is not needed, but whatever. 
+//Radio stuff
+RH_ASK driver(2000, 7, LED_PIN, BUZZER_PIN);
```
Comment (radio-satellites) [inline firmware_no_aux.cpp]: @TheBoss313 Sorry for my poor documentation. The final argument (PTT) probably should be an unused pin (i.e 4), not the buzzer pin.

Comment (radio-satellites) [reply firmware_no_aux.cpp]: I'm juggling between a custom modulation (CW, the morse section) and ASK packets via the RadioHead library

Comment (radio-satellites) [reply firmware_no_aux.cpp]: Probably I'll switch to something else soon. Leave this for now though

Comment (radio-satellites) [reply firmware_no_aux.cpp]: AHHHHHHHHHHHH this is the no_aux file

Sorry!!!!

File: firmware.CPP [hunk 2]
```diff
@@ -23,225 +25,220 @@ Adafruit_NeoPixel pixels(NUMPIXELS, PIN, NEO_GRB + NEO_KHZ800);
...
+    while (*str)
+        send(*str++);
+    Serial.println("");
 }
 
 
-//Radio shtuff
-RH_ASK driver(2000, 7, 12, 5); //Which will initialise the driver at 2000 bps, recieve on GPIO7, transmit on GPIO12, PTT on GPIO5. This is very poorly documented. PTT is not needed, but whatever. 
+//Radio stuff
+RH_ASK driver(2000, 7, LED_PIN, BUZZER_PIN);
```
Comment (radio-satellites) [inline firmware.CPP]: @TheBoss313 So the receive command pin is unused really

Comment (radio-satellites) [reply firmware.CPP]: @TheBoss313 Sorry for my poor documentation. The final argument (PTT) probably should be an unused pin (i.e 4), not the buzzer pin.

Comment (radio-satellites) [reply firmware.CPP]: I'm juggling between a custom modulation (CW, the morse section) and ASK packets via the RadioHead library

Comment (radio-satellites) [reply firmware.CPP]: Probably I'll switch to something else soon. Leave this for now though, could you please change the PTT pin though>

**Comment by radio-satellites** [comment]
Thank you so much for the cool PR!!

**Comment by radio-satellites** [comment]
There's just a tiny problem in the PTT function and then it's perfect!!

I'll run it on the hardware later today.

Thanks!!!

File: firmware.CPP (hunk 2 above)
Comment (Vlad-Galeev) [reply firmware.CPP]: Sure, will do

**Comment by Vlad-Galeev** [comment]
Fixed the pin to use pin 4

File: firmware.CPP (hunk 2 above)
Comment (radio-satellites) [reply firmware.CPP]: What I meant was something like RH_ASK driver(2000, 7, 12, 1); perhaps that might work better...

Comment (Vlad-Galeev) [reply firmware.CPP]: Using pins 0 and 1 on Arduino is a bad idea if you plan to use the USB port, as the communication happens to be on those pins

**Pull request merged**